### PR TITLE
[REVIEW] Adding the original source for Airline Data

### PR DESCRIPTION
**Pull request by Nanthini10**
Changing README to include the original dataset for airline

### CODE REVIEW

**Comment by drobison00** [comment]
@Nanthini10 Is this still relevant? Does it need to be merged?

**Comment by Nanthini10** [comment]
@drobison00 Yes, this is still relevant! I don't have merge access, can you do it?

File: README.md [hunk 1]
```diff
@@ -9,6 +9,26 @@ This repository provides example notebooks and "getting started" code
 samples to help you integrate RAPIDS with the hyperparameter
 optimization services from Azure ML, AWS Sagemaker, Google
 Cloud, and Databricks. The directory for each cloud contains a step-by-step guide to
+launch an example hyperparameter optimization job.
+
+Each example job will use RAPIDS
```
[the remark this answers is not in the document]
Comment (drobison00) [reply README.md:14]: Does this paragraph need to move into a subdirectory?

File: README.md [hunk 2]
```diff
@@ -9,6 +9,26 @@ This repository provides example notebooks and "getting started" code
...
+[cuDF](https://github.com/rapidsai/cudf) to load and preprocess 20
+million rows of airline arrival and departure data and build a model
+to predict whether or not a flight will arrive on time (dataset is sourced from [Airline On-Time Performance Data from BTS](https://transtats.bts.gov/Tables.asp?DB_ID=120&DB_Name=Airline%20On-Time%20Performance%20Data&DB_Short_Name=On-Time#)). It
+demonstrates both [cuML](https://github.com/rapidsai/cuml) Random
+Forests and GPU-accelerated XGBoost modeling.
+
+## Microsoft Azure ML
```
Comment (drobison00) [inline README.md:21]: I think all these ## <TOPIC> blocks are out of data as well, and can probably be removed. Looks like some extra stuff sneaked in with the merge.

**Comment by Nanthini10** [comment]
@drobison00  - Sorry for the confusion, the updated README doesn't look like it needs the data source to be used. I'll include the data source in the notebook instead, that'll be cleaner. Closing this issue

**Pull request closed without merging**